### PR TITLE
Removed eslint-config-node-services and mocha-eslint. 

### PR DESCRIPTION
**Pull request by hknustwmf**
Also removed nsp since apparently it [has been shutdown (StackOverflow)](https://stackoverflow.com/questions/53716991/node-security-service-shutdown-getaddrinfo-enotfound-api-nodesecurity-io) and now throws an exception. 
+) Client request error: getaddrinfo ENOTFOUND api.nodesecurity.io api.nodesecurity.io:443

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -71,8 +71,8 @@ function getOptions(uri, o, method) {
         }
     }
 
-    if ((o.method === 'get' || o.method === 'put')
-        && o.retries === undefined) {
+    if ((o.method === 'get' || o.method === 'put') &&
+        o.retries === undefined) {
```
Comment (Pchelolo) [inline index.js]: One more indent for better readability please

File: index.js [hunk 2]
```diff
@@ -89,8 +89,8 @@ function getOptions(uri, o, method) {
         o.timeout = 2 * 60 * 1000; // 2 minutes
     }
 
-    if ((o.headers && /\bgzip\b/.test(o.headers['accept-encoding']))
-        || (o.gzip === undefined && o.method === 'get')) {
+    if ((o.headers && /\bgzip\b/.test(o.headers['accept-encoding'])) ||
+        (o.gzip === undefined && o.method === 'get')) {
```
Comment (Pchelolo) [inline index.js]: Idem

File: index.js [hunk 3]
```diff
@@ -174,8 +173,8 @@ class Request {
             }
 
             // 204, 205 and 304 responses must not contain any body
-            if (response.statusCode === 204 || response.statusCode === 205
-                || response.statusCode === 304) {
+            if (response.statusCode === 204 || response.statusCode === 205 ||
+                response.statusCode === 304) {
```
Comment (Pchelolo) [inline index.js]: Idem

File: index.js [hunk 4]
```diff
@@ -191,8 +190,8 @@ class Request {
                 origURI += `?${querystring.stringify(this.options.qs)}`;
             }
 
-            if (origURI !== response.request.uri.href
-                && url.format(origURI) !== response.request.uri.href) {
+            if (origURI !== response.request.uri.href &&
+                url.format(origURI) !== response.request.uri.href) {
```
Comment (Pchelolo) [inline index.js:193]: Idem

**Comment by Pchelolo** [comment]
LGTM, couple of minor nits regarding conditionals indentation. Desnt seem like therese a way to enforce it, but please double check, I might have missed it

File: index.js (hunk 1 above)
Comment (hknustwmf) [reply index.js]: Actually fits on 1 line

File: index.js (hunk 2 above)
Comment (hknustwmf) [reply index.js]: Done

File: index.js (hunk 4 above)
Comment (hknustwmf) [reply index.js:193]: Done

**Pull request merged**